pkg/minikube/update/update_test.go

```diff
@@ -1,183 +0,0 @@
-/*
```

Comment:
Yes, having unused code doesn't make sense and we can add again once we have functionality with us.